### ISSUE TITLE
multi-file upload request

### ISSUE THREAD

**Issue by den-run-ai** [peteut/ipython-file-upload#5]
http://stackoverflow.com/questions/31643477/how-to-make-multi-file-upload-widget-in-ipython-notebook

**Comment by peteut** [comment]
Yes, multi-file support is a somewhat mandatory feature. I'll have a look at it, thanks!
